materialsLibrary/test/index.html

```diff
@@ -144,7 +144,7 @@
 				simple.diffuseTexture.uScale = 5;
 				simple.diffuseTexture.vScale = 5;
 				
-				var water = new BABYLON.WaterMaterial("water", scene, currentMesh);
+				var water = new BABYLON.WaterMaterial("water", scene);
 				water.backFaceCulling = false;
 				water.enableRenderTargets(false);
```

Comment:
Oh, the functon definition is:
    public enableRenderTargets(enable: boolean): void;
true to enable and false to disable.
It is enabled by default